packages/node/src/index.ts

```diff
@@ -381,7 +381,10 @@ export const build: BuildV3 = async ({
 };
 
 export const prepareCache: PrepareCache = async ({ workPath }) => {
-  const cache = await glob('node_modules/**', workPath);
+  const cache = {
+    ...(await glob('node_modules/**', workPath)),
+    ...(await glob('**/node_modules/**', workPath)),
```

Comment:
Same for this glob, can it be a single pattern to match both cases?

Reply:
I think we need to consider `repoRootPath` here, when it's set.